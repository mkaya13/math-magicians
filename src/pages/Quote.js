import React from 'react';
import '../index.css';

const Home = () => (
  <div className="quote-page">
    <p className="quote">
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding. -William Paul Thurston
    </p>
  </div>
);
export default Home;
