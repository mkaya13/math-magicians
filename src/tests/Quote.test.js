import TestRenderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Quote from '../pages/Quote';

describe('Test Quote component', () => {
  it('should renders Quote component', () => {
    const QuoteComponent = TestRenderer.create(<Router><Quote /></Router>).toJSON();
    expect(QuoteComponent).toMatchSnapshot();
  });
});
