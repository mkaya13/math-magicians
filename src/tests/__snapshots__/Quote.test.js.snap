// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quote component should renders Quote component 1`] = `
<div
  className="quote-page"
>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
